serverless/serverless
# fix: openwhisk-java-maven template, and corresponding docs

## What did you implement



Fixed: #7160 by adding the JAR file in `handler` field of `serverless.yml`.
Also added `package.json` in the template directory to be used by `npm install` to install provider plugin.
Besides, added docs of a simple java example with openwhisk (the same as the simple project in the template directory) in `docs/providers/openwhisk/examples`.

**However, the URL links in `docs/providers/openwhisk/examples/README.md` and `docs/providers/openwhisk/examples/hello-world/java/README.md` pointing to reference pages in serverless.com are now invalid, as I don't have access to adding pages to the website.**

## How can we verify it


Follow the steps in `docs/providers/openwhisk/examples/hello-world/java/README.md`, you should observe the expected outputs as in the doc.

Before the changes, you'll get:
```
  Serverless Error ---------------------------------------

  Function handler (com.example.FunctionApp) does not exist.
```
at the step of `sls deploy`

## Todos

**_Is this ready for review?:_** YES
**_Is it a breaking change?:_** NO
